src/renderer/patches/keybinds.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0
+ * Vesktop, a desktop app aiming to give you a snappier Discord Experience
+ * Copyright (c) 2023 Vendicated and Vencord contributors
+ */
+
+import { keybindCallbacks } from "renderer";
+
+import { addPatch } from "./shared";
+
+addPatch({
+    patches: [
+        {
+            find: ".default.Messages.KEYBINDS,children:",
+            replacement: {
+                match: /.\.isPlatformEmbedded/,
+                replace: "true"
+            }
+        },
+        {
+            find: "[kb store] KeybindStore",
+            replacement: {
+                match: /(inputEventRegister\(parseInt\((.{1,2})\),(.{1,2}),(.{1,2}),(.{1,2})\);else\{)([^;]*;[^;]*;.{1,2}\.keyup&&.{1,2}\.bindGlobal\(\(0,(.{1,2}\.toString)\))/,
+                replace: "$1$self.registerKeybind($2,$3,$4,$7);return;$6"
+            }
+        },
+        {
+            find: "[kb store] KeybindStore",
+            replacement: {
+                // WHY IS THE RADIX SPEICIFIED
```

Comment:
Just FYI I know why the radix is specified now

From https://github.com/denysdovhan/wtfjs#parseint-is-a-bad-guy:

> 💡 Explanation: If the input string begins with "0", radix is eight (octal) or 10 (decimal). Exactly which radix is chosen is implementation-dependent. ECMAScript 5 specifies that 10 (decimal) is used, but not all browsers support this yet. For this reason always specify a radix when using parseInt.